core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -104,10 +125,30 @@
     word-wrap: break-word;
   }
   .oppia-text-to-translate-container {
-    border: 1px solid #ccc;
+    background-color: rgb(233, 247,246);
     margin: 15px 0;
+    margin-bottom: 0;
+    padding: 20px;
+  }
+  .oppia-machine-translation-container {
+    background-color: rgba(251, 251, 251);
```

Comment:
The background color is a bit light—hard to tell (on my screen at least) that there's a box there. Maybe try the light gray listed in https://docs.google.com/document/d/1R9PWRBA7yPCiHxc325gx1xHd_RMLRlCjMFWDCowUPag/edit#heading=h.kwpmfhhieo37?